templates/base.html

```diff
@@ -56,6 +56,10 @@
                     {%- include 'menu.html' -%}
                 {%- endif -%}
                 <div class="off-canvas-content" data-off-canvas-content>
+                    <div class="theme-toggle">
+                        <i class="theme fa fa-moon-o to-dark fa-2x" onclick="toggleTheme()"></i> 
```

Comment:
Don't bind click's straight away, bind them using javascript when the document's ready.

Reply:
Okay, I will work on this document.ready() and update this PR.